core/src/main/java/hudson/tasks/_maven/Maven3MojoNote.java

```diff
@@ -65,5 +66,6 @@ public String getDisplayName() {
         }
     }
 
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
```

Comment:
Should it actually be final? If no, some justification would be cool

Reply:
I guess noone will try to modify the pattern with script console, so I'm setting it final.